web-ui/src/components/feedback_request_card/FeedbackRequestCard.jsx

```diff
@@ -19,6 +18,8 @@ import Divider from '@mui/material/Divider';
 import Button from '@mui/material/Button';
 import queryString from 'query-string';
 
+import ExpandMore, { ExpandMoreIcon } from '../expand-more/ExpandMore';
```

Comment:
Is `ExpandMoreIcon` still needed in this file?

Reply:
Initially it was provided as a convenience to allow users to customize the contents of the `button` element. I've since updated the code to allow for self-closing of the ExpandMore and removed the button export. Now when the ExpandMore tag is self-closing, the ExpandMoreIcon will automatically be selected for the user.